tests/e2e/e2e_setup_test.go

```diff
@@ -59,7 +59,7 @@ var _ = Describe("Cluster preparation", labels.Feature.Cluster, func() {
 						"--claim-mapping-username-claim", "email",
 						"--claim-mapping-groups-claim", "groups",
 						"--console-client-id", "8a769b34-13c9-4f5b-9933-ec439700ec67",
-						"--console-client-secret", "vfq8Q~XpgXx9vsKF~XSW1bcSowfJP2JGraybYa7X",
+						"--console-client-secret", "xxxxx~xxxxxxxxxx~xxxxxxxxxxxxxxxxxxxxxxx",
```

Comment:
Will it work? Why not generate a random value write to a file and mask it in the command line?

Reply:
It can work.Here is the [log](https://privatebin.corp.redhat.com/?205ccfbf365bea65#5rxHCEmdN3iWpppGMdNCpq4uJukHkwqv6TPWd9tYEKdz). Let me check how to generate one like <xxxxx>~<xxxxxxxxxx >~<xxxxxxxxxxxxxxxxxxxxxxx>